azurerm/helpers/azure/eventhub.go

```diff
@@ -100,7 +100,13 @@ func EventHubAuthorizationRuleSchemaFrom(s map[string]*schema.Schema) map[string
 			Default:  false,
 		},
 
-		"primary_key": {
+		"alias_primary_connection_string": {
```

Comment:
could we rename this to 
```suggestion
		"primary_connection_string_alias": {
```

Reply:
updated